.eslintrc

```diff
@@ -36,6 +36,7 @@
     "guard-for-in": 0,
     "jsx-quotes": 1,
     "no-bitwise": 0,
+    "no-console": 0,
```

Comment:
This seems _dangerous_ to me. I'd at least go for warning otherwise we might accidentally commit logging statements. One can also conditionally allow `.warn()` and `.error()` logs if that's what we want.

Reply:
Thanks: I've changed it so we only allow `console.warn()` and `console.error()`, but `log` and `info` will not be allowed. Longer term, we want a logging framework, but this is good enough for now?